DiamondLightSource/SynchWeb
# [LIMS-832]Feature: Use shipping service for incoming shipments

Requires #618 

JIRA ticket: [LIMS-832](https://jira.diamond.ac.uk/browse/lims-832)

Changes:
- Add config parameter `$use_shipping_service_incoming_shipments` to toggle using the shipping service for incoming shipments
- Make requests to the shipping service for booking incoming shipments
- Fix problem logging `$shipment_data` on shipping service dispatch error
- Format address and post code when first loading the dispatch form

To test:
- Check that, with  `$use_shipping_service_incoming_shipments=true;`, shipments are created/updated in the shipping service when booking an incoming shipment
- Check that SynchWeb behaviour is unchanged when booking an incoming shipment
